dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.mpFaultTolerance-telemetry-metrics2.0.feature

```diff
@@ -0,0 +1,9 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpFaultTolerance-telemetry-metrics2.0
```

Comment:
I suggest that this should be named
`io.openliberty.mpFaultTolerance41-mpTelemetry20`